### PR TITLE
fix css bugs

### LINKED ISSUE #3403: Some bugs in 1.6.2 release
Posting a list of bugs that I noticed during testing the 1.6.2 release:

1. Buttons alignment is a bit off in the launcher view. We decided to use the right-alignment in the wallet, so let's change it here as well. Also, please, swap the buttons, it should be Cancel-Continue if reading from left to right.
image
image

2. Arrow within dropdowns has to be turned up when clicked
image
image

3. Same issue with buttons alignment(move and swap):
image

4. Swap the buttons (for all pop ups that have buttons):
image
image
image
image
image

5. Add a spacing here:
image

6. Move the button to the right:
image

Also @linnutee you mentioned something about icons and gifs on loader. I didn't notice anything weird, so could you explain here? Thanks.

### PR DESCRIPTION
**Pull request by bgptr**
Closes #3403

### CODE REVIEW

File: app/style/ReactSelectGlobal.css [hunk 1]
```diff
@@ -226,7 +226,6 @@
   height: 0;
   width: 0;
   position: relative;
-  transition: all var(--input-transition-time) var(--input-timing-function);
```
Comment (amassarwi) [inline app/style/ReactSelectGlobal.css:229]: are these used in other places 🤔 ?

**Review by amassarwi — APPROVED** [review]
LGTM + 1 question.

File: app/style/ReactSelectGlobal.css (hunk 1 above)
Comment (bgptr) [reply app/style/ReactSelectGlobal.css:229]: @amassarwi Thanks for the review. 
This transition only affects the arrow icon. I've removed the line for the sake of uniformity. Other similar arrows aren't animated in the app.

Comment (amassarwi) [reply app/style/ReactSelectGlobal.css:229]: i mean the theme variables: `input-transition-time` & `input-timing-function` - i'm not sure if they are used/needed in other places, if not we could ditch them from the theme files.

Comment (bgptr) [reply app/style/ReactSelectGlobal.css:229]: Ahh, I see :) They are used in this file, but ditching them hasn't changed the behavior of the select control.

Comment (amassarwi) [reply app/style/ReactSelectGlobal.css:229]: Noice, thanks 🥇

**Review by alexlyp — APPROVED** [review]

**Pull request merged**